src/app.md

```diff
@@ -119,7 +119,22 @@ Your favorite number must be 256.
 
 [![log-badge]][log] [![env_logger-badge]][env_logger] [![cat-debugging-badge]][cat-debugging]
 
-[Write me!](https://github.com/brson/rust-cookbook/issues/61)
+```rust
+#[macro_use] extern crate log;
+extern crate env_logger;
+
+fn main() {
+    env_logger::init().unwrap();
```

Comment:
This is fine and the env_logger docs recommend unwrap. It fails if you register two loggers but that never happens in any reasonable program.